spec/ics-24-host-requirements/README.md

```diff
@@ -0,0 +1,103 @@
+---
+ics: 24
+title: Host State Machine Requirements
+stage: draft
+category: ibc-core
+requires: 2
+required-by: 2, 3, 4, 5, 18
+author: Christopher Goes <cwgoes@tendermint.com>
+created: 2019-04-16
+modified: 2019-04-29
+---
+
+# Synopsis
+
+This specification defines the minimal set of interfaces which must be provided and properties which must be fulfilled by a blockchain & state machine hosting an IBC handler (implementation of the interblockchain communication protocol).
```

Comment:
Should this link to the architecture document here to have quick access to where the IBC handler fits in the larger architecture (and vice versa?).

Reply:
aye, added.